.storybook/preview-head.html

```diff
@@ -0,0 +1,12 @@
+<style>
+    body {
+        font-family: 'Fira Mono', monospace;
+        color: '#fff';
```

Comment:
I think the storybook html/body styles should match what we have in `layout.js`

This color isn't working because of the quotes.